c_common/models/live_packet_gather/src/live_packet_gather.c

```diff
@@ -450,64 +447,45 @@ static bool configure_sdp_msg(void) {
     temp_header |= config.payload_timestamp << PAYLOAD_IS_TIMESTAMP;
     temp_header |= config.packet_type << PACKET_TYPE;
 
-    header_len = 2;
-
     // pointers for AER packet header, prefix and data
-    void *temp_ptr;
```

Comment:
We don't need this. We can just use `sdp_msg_aer_data` and bump it along as required.

Reply:
yep